profiles/config.py

```diff
@@ -9,7 +9,7 @@
 class Config(ABC):
     name = 'unknown'
     DEBUG = False
-    SQLALCHEMY_COMMIT_ON_TEARDOWN = True
+    SQLALCHEMY_COMMIT_ON_TEARDOWN = False
```

Comment:
However if this is turned to `False` there are probably many places where we need to commit the transaction manually.
The pattern is the [Unit Of Work](https://martinfowler.com/eaaCatalog/unitOfWork.html), SQLAlchemy implements it with `db.session`, but perhaps we can automate the calls to that with some decorator (magic word) to put around the routes?